package.json

```diff
@@ -17,7 +17,7 @@
     "kubernetes-object-describer": "1.1.4",
     "kubernetes-topology-graph": "0.0.23",
     "moment": "2.10.6",
-    "mustache": "0.8.2",
+    "mustache": "2.2.1",
```

Comment:
Let's not bother with updating to a incompatible version and quote all attributes instead, which seems the right thing to do in any case, even for newer Mustache versions.